starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -395,9 +401,25 @@ def main():
   def IsSbSymbol(symbol):
     return symbol.startswith('Sb') or symbol.startswith('kSb')
 
+  # Allowed POSIX symbols in Starboard 16
+  allowed_sb16_posix_symbols = [
+      'calloc',
+      'free',
+      'malloc',
+      'posix_memalign',
+      'realloc',
+  ]
+
+  def IsAllowedPosixSybol(symbol, sb_api_version):
+    if int(sb_api_version) == 16:
```

Comment:
If you're only using sb_api_version as an int, then make the argument type an int instead of string